dynamicscrm/connector.go

```diff
@@ -64,6 +61,20 @@ func (c *Connector) getURL(arg string) (*urlbuilder.URL, error) {
 	return constructURL(c.BaseURL, apiVersion, arg)
 }
 
+func (c *Connector) getEntityDefinitionURL(arg naming.SingularString) (*urlbuilder.URL, error) {
```

Comment:
Can you please add comments above `getEntityDefinitionURL` just like you have for `getEntityAttributesURL` describing what it does?